lib/huginn_agent/patches/coverage.rb

```diff
@@ -0,0 +1,19 @@
+require 'simplecov'
+
+if !ENV['COVERAGE']
+  require 'coveralls'
+  module Coveralls
+    module Configuration
+      def self.root
+	File.expand_path(File.join(Dir.pwd, '../..'))
```

Comment:
Actually the paths need to be relative to the current working directory. The file is required by rspec which is running in `somepath/agent_gem/spec/huginn` and we need the root to be `somepath/agent_gem`. Using `__FILE__` would result in a path somewhere in the users gem directory :)